README.md

```diff
@@ -6,22 +6,43 @@
 2. Describe why a virtual DOM might be useful
 3. Describe some of the tradeoffs of using a virtual DOM
 
-## Overview
-
-We want to give students a very high-level look at virtual DOMs in the abstract.
-It's totally okay to focus on React, but students should understand that React's
-isn't the only virtual DOM implementation in town, and they should also
-understand that React is more than simply a virtual DOM (even if that's one of
-its core ideas that it happens to execute very well).
-
-It's worth mentioning some of the tradeoffs w/r/t to reconciling virtual and
-actual state. Inputs are a bit wonky (but more on that in a later lesson, of
-course), and side effects in when rendering can grind an application to a halt.
-
-At the same time, JavaScript is fast while the DOM APIs are slow — this point is
-worth emphasizing.
+## What's a virtual DOM?
+![React VR?](https://media.giphy.com/media/3o7qDL7l1IZAQvxvj2/giphy.gif)
+
+By now, you should already know what the DOM is: a programmatic representation of the document we see in the browser.
+So, what's all this virtual DOM ruckus about? Well, it turns out that our good 'ol friend DOM is slow — terribly slow. He
+makes a granny using a walker look like Usain Bolt.
+
+Virtual DOM is a technique employed by several front-end libraries and frameworks, most notably React. In a nutshell,
+virtual DOM builds a _virtual_ representation of what our document should look like. When we're ready to render things
+to the screen, the virtual DOM will take a look at the existing DOM and change only what needs to be changed (in more
+technical terms, it's diffing and re-rendering the changes).
+
+The performance gains here are not to be underestimated, due to the way the virtual DOM works. Virtual DOM is created by
```

Comment:
This sentence confuses me a bit: "The performance gains here are not to be underestimated, due to the way the Virtual DOM works." Seems like you are saying that due to the implementation we shouldn't underestimate. I think just: "The performance gains here are not to be underestimated" will do well! And you are right!